### PR TITLE
Roadmap update to match #575 and more

### PR DESCRIPTION
**Pull request by el00ruobuob**
This is ~~two~~ four commits:
- One for the French Roadmap to be updated according to @mattcode55 #575 (in the hope @rehrar and him would catch and merge it first), along with a few correction and a general change from "hardfork" to "network upgrade";
- One for the early 2018 achievements (still in French).
- One for fixing stuff after a build test (**new 05/11**)
- One for updating all translations at the same level FR was in the second commit. (**new 05/11**)

Regarding the early 2018 achievements, these is the English version, ~~for review comodities~~ that is visible in the code. @erciccione @zofiazinha @hrumag those needs to be updated for Polish and Italian (again, i can update for you if it is easier and you provide the translation):

- name: New Proof of Work CryptoNoteV7
  date: March, 2018
  status: completed
- name: Network upgrade to increase minimal ringsize to 7, integrate multisig, subaddresses, and change PoW algo.
  date: 2018-04-04
  status: completed
- name: Getmonero.org Localization in French and Polish
   date: 2018-04-24
   status: completed

~~Once #575 is merged, i will work~~ I have worked on a general roadmap update with those listed elements:
- Update of roadmap.md for Polish & Italian too ;
- s/HardFork/Network Upgrade/ ;
  - "aggiornamento della rete" in Italian
  - @zofiazinha what's your Polish version?
- Early 2018 achievments ;
- Update ~~of~~ on all ~~untranslated and template~~ .yml files ~~_only_~~.

So @erciccione, you see i can still found something to do ;)

### CODE REVIEW

**Comment by el00ruobuob** [comment]
French part to close #572

**Comment by el00ruobuob** [comment]
Successfully tested:

  | Edge | IE 11 | Chrome | Firefox | Opera | Vivaldi
-- | -- | -- | -- | -- | -- | --
Windows 7 | - | OK | OK | OK | OK | OK
Windows 10 | OK | OK | OK | OK | OK | OK
Ubuntu 18.04 | - | - | OK | OK | OK | OK
CentOS 7 | - | - | - | OK | OK | OK

**Comment by el00ruobuob** [comment]
I have rebased and update everything.
All is working fine here:

  | Edge | IE 11 | Chrome | Firefox | Opera | Vivaldi
-- | -- | -- | -- | -- | -- | --
Windows 7 | - | OK | OK | OK | OK | OK
Windows 10 | OK | OK | OK | OK | OK | OK
Ubuntu 18.04 | - | - | OK | OK | OK | OK
CentOS 7 | - | - | - | OK | OK | OK

@rehrar #575 has updated CSS which is "braking" the roadmap page (year header place divided by 6 against 5 before, so there's an empty space on the right).
Also a glitch of both 2017 and 2018 being checked.
This one correct those both concerns.

**Comment by zofiazinha** [comment]
name: New Proof of Work CryptoNoteV7 -> **Nowy dowód pracy CryptoNoteV7**
  
    name: Network upgrade to increase minimal ringsize to 7, integrate multisig, subaddresses, and change PoW algo. -> **Aktualizacja sieci w celu zmniejszenia minimalnego rozmiaru pierścienia do 7, zintegrowania multipodpisu i podadresów oraz zmiany algorytmu dowodu pracy.**

    name: Getmonero.org Localization in French and Polish -> **Lokalizacja strony Getmonero.org na język francuski i polski.**

s/HardFork/Network Upgrade/ ;

    "aggiornamento della rete" in Italian
    @zofiazinha what's your Polish version? -> **Aktualizacja sieci**

**Comment by el00ruobuob** [comment]
Thanks @zofiazinha! I have updated it.

Pinging reviewers @mattcode55 @MaxXor

File: _data/lang/en/roadmap.yml [hunk 1]
```diff
@@ -86,6 +86,15 @@
       status: completed
 - year: 2018
   accomplishments:
+    - name: New Proof of Work CryptoNoteV7
+      date: March, 2018
+      status: completed
+    - name: Network upgrade to increase minimal ringsize to 7, integrate multisig, subaddresses, and change PoW algo.
```
Comment (MaxXor) [inline _data/lang/en/roadmap.yml]: Please remove the `dot .` at the end, it's no sentence. Also you can change the date of the item "New Proof of Work CryptoNoteV7" to the same date as the network upgrade.

**Comment by MaxXor** [comment]
Other than the thing I've mentioned it's good to go. When you are done, squash the commits into one and it can get merged.

File: _data/lang/en/roadmap.yml (hunk 1 above)
Comment (el00ruobuob) [reply _data/lang/en/roadmap.yml]: ok

Comment (el00ruobuob) [reply _data/lang/en/roadmap.yml]: Btw i just saw i wrote 4/4 where the network upgrade occurs on the 6th

**Comment by el00ruobuob** [comment]
Squashed it is.

**Comment by el00ruobuob** [comment]
@erciccione or @hrumag do you want me to include the Italian for early 2018, as i did for polish? Otherwise i'll get in touch with @rehrar and @luigi1111 in order to include this in the next site build.

**Comment by el00ruobuob** [comment]
Rebased, test in progress.

**Comment by el00ruobuob** [comment]
@rehrar @luigi1111 Tested ok (after a glitch correction) on the following platforms:

  | Edge | IE 11 | Chrome | Firefox | Opera | Vivaldi
-- | -- | -- | -- | -- | -- | --
Windows 7 | - | OK | OK | OK | OK | OK
Windows 10 | OK | OK | OK | OK | OK | OK
Ubuntu 18.04 | - | - | OK | OK | OK | OK
CentOS 7 | - | - | - | OK | OK | OK

**Comment by el00ruobuob** [comment]
On last glitch corrected: the status on top wasn't using the new reimplemented localization.
Before:
image

Relocalized:
image

After:
image

**Comment by el00ruobuob** [comment]
@MaxXor Corrected:
image

**Comment by el00ruobuob** [comment]
ping @rehrar & @luigi1111 Could we merge this? Or do you want more approvers?

**Comment by luigi1111** [comment]
Fine to merge after next rebase.

**Comment by el00ruobuob** [comment]
Rebased it is @luigi1111.

**Pull request merged**